v20.2/architecture/replication-layer.md

```diff
@@ -79,6 +79,21 @@ A table's meta and system ranges (detailed in the [distribution layer](distribut
 
 However, unlike table data, system ranges cannot use epoch-based leases because that would create a circular dependency: system ranges are already being used to implement epoch-based leases for table data. Therefore, system ranges use expiration-based leases instead. Expiration-based leases expire at a particular timestamp (typically after a few seconds). However, as long as a node continues proposing Raft commands, it continues to extend the expiration of its leases. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
 
+#### How leases are transferred from a dead node
+
+When a node disconnects, the process by which each of its leases is transferred to a healthy node is as follows:
+
+1. The cluster waits 9 seconds for the dead node's liveness record, which is stored in a system range, to expire.
+2. A healthy node attempts to acquire the lease. This is rejected because lease acquisition can only happen on the Raft leader, which the healthy node is not (yet).  Therefore, an election must be held.
+3. The rejected attempt at lease acquisition unquiesces the system range holding the liveness record.
```

Comment:
The range that is unquiesed is the range for which we are trying to acquire a lease which could be the liveness range if that was on the disconnected node.

Reply:
Updated to "The rejected attempt at lease acquisition unquiesces the range associated with the lease.  This range could be the one holding the liveness record, if that was on the disconnected node."